### PR TITLE
feat(profiling): Render flamegraph in transaction summary

### PR DESCRIPTION
**Pull request by Zylphrex**
As part of the effort to make the profiling pages compatible with both continuous profiling and transaction profiling, this moves the flamegraph from the profile summary page into a tab of the transaction summary page.

### CODE REVIEW

File: static/app/views/performance/transactionSummary/transactionProfiles/index.tsx [hunk 1]
```diff
@@ -125,11 +153,339 @@ function Profiles(): React.ReactElement {
...
+      fillSpace
+      childComponent={() => {
+        return (
+          <StyledMain fullWidth>
+            <FilterActions>
+              <PageFilterBar condensed>
+                <EnvironmentPageFilter />
+                <DatePageFilter />
+              </PageFilterBar>
+              {/* TODO: This doesn't actually search anything
```
Comment (JonasBa) [inline static/app/views/performance/transactionSummary/transactionProfiles/index.tsx:274]: Should we just remove this?

File: static/app/views/performance/transactionSummary/transactionProfiles/index.tsx [hunk 2]
```diff
@@ -125,11 +153,339 @@ function Profiles(): React.ReactElement {
...
+  //   c.setFilterValues('transaction', [transaction]);
+
+  //   Object.keys(c.filters).forEach(field => {
+  //     if (isAggregateField(field)) {
+  //       c.removeFilter(field);
+  //     }
+  //   });
+  // }, [transaction, query]);
+
+  // const handleSearch: SmartSearchBarProps['onSearch'] = useCallback(
```
Comment (JonasBa) [inline static/app/views/performance/transactionSummary/transactionProfiles/index.tsx:201]: same q here

File: static/app/views/performance/transactionSummary/transactionProfiles/index.tsx [hunk 3]
```diff
@@ -125,11 +153,339 @@ function Profiles(): React.ReactElement {
...
+  organization: Organization;
+  transaction: string;
+}
+
+function Profiles({organization, transaction}: ProfilesProps) {
+  const location = useLocation();
+  const projects = useProjects();
+  const {selection} = usePageFilters();
+
+  // const query = decodeScalar(location.query.query, '');
```
Comment (JonasBa) [inline static/app/views/performance/transactionSummary/transactionProfiles/index.tsx:187]: We can just delete this I guess?

File: static/app/views/performance/transactionSummary/transactionProfiles/index.tsx [hunk 4]
```diff
@@ -125,11 +153,339 @@ function Profiles(): React.ReactElement {
...
+
 const FilterActions = styled('div')`
   margin-bottom: ${space(2)};
   gap: ${space(2)};
   display: grid;
   grid-template-columns: min-content 1fr;
 `;
 
-export default Profiles;
+// const StyledSearchBar = styled(SearchBar)`
```
Comment (JonasBa) [inline static/app/views/performance/transactionSummary/transactionProfiles/index.tsx:418]: 🔪 ?

**Review by JonasBa — APPROVED** [review]

File: static/app/views/performance/transactionSummary/transactionProfiles/index.tsx (hunk 1 above)
Comment (Zylphrex) [reply static/app/views/performance/transactionSummary/transactionProfiles/index.tsx:274]: I'm going to follow up with this soon

**Review by JonasBa — APPROVED** [review]

**Pull request merged**